lib/alephant/broker/response/base.rb

```diff
@@ -21,11 +21,15 @@ class Base
 
         def initialize(status = 200, content_type = "text/html")
           @content = STATUS_CODE_MAPPING[status]
-          @headers = { "Content-Type" => content_type }
-          @headers.merge!(Broker.config[:headers]) if Broker.config.has_key?(:headers)
+          @headers = {
+            "Content-Type"                  => content_type,
+            "Access-Control-Allow-Headers"  => "If-None-Match"
+          }
+          headers.merge!(Broker.config[:headers]) if Broker.config.has_key?(:headers)
           @status  = status
 
           add_no_cache_headers if should_add_no_cache_headers?(status)
+          add_etag_allow_header if headers.has_key?(:ETag)
```

Comment:
The header gets returned as a symbol from the file:
> https://github.com/BBC-News/alephant-storage/blob/master/lib/alephant/storage.rb#L78

But, it does look like we convert to a string when we start processing the component headers:
> https://github.com/BBC-News/alephant-broker/blob/master/lib/alephant/broker/component.rb#L48